### PR TITLE
format error

### PR DESCRIPTION
**Pull request by timchenxiaoyu**
change isntall docs

### CODE REVIEW

**Comment by giuseppe** [comment]
what is wrong with the original version?

File: docs/install/Fedora.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ dnf install atomic
 ##Make
 On Fedora, You need to install required build dependencies
 ```
-yum-builddep atomic
+#yum-builddep atomic
```
Comment (rhatdan) [inline docs/install/Fedora.md]: Should probably change this to 

dnf builddep atomic

Since yum is going away on Fedora

**Comment by rhatdan** [comment]
bot, add author to whitelist

File: docs/install/Fedora.md (hunk 1 above)
Comment (timchenxiaoyu) [inline docs/install/Fedora.md]: I install  stomic on centos7  use yum install atomic

Comment (timchenxiaoyu) [inline docs/install/Fedora.md]: I complie atomic on centos7 error 
W: 91,12: __init__ method from a non direct base class 'super' is called (non-parent-init-called)
W: 86, 8: __init__ method from base class 'Action' is not called (super-init-not-called)
W:136,11: Except doesn't do anything (pointless-except)
W:166,11: Except doesn't do anything (pointless-except)
ERROR:root:Could not find any typelib for OSTree

Comment (rhatdan) [inline docs/install/Fedora.md]: Your patch is to the fedora.md,  If we have a centos.md, we should put your changes their.

**Comment by rhatdan** [comment]
@timchenxiaoyu Any response?

File: docs/install/Fedora.md (hunk 1 above)
Comment (timchenxiaoyu) [inline docs/install/Fedora.md]: sorry。I am very busy recently。i will contine test lataer。I want to run docker in special independent dir，could you teach me，how to slove this，for example，docker run --dir

Comment (timchenxiaoyu) [inline docs/install/Fedora.md]: sorry。I am very busy recently。i will contine test lataer。I want to run docker in special independent dir，could you teach me，how to slove this，for example，docker run --dir

Comment (rhatdan) [inline docs/install/Fedora.md]: Please send email to dwalsh@redhat.com and I can help you.  I am closing this issue.

Comment (timchenxiaoyu) [inline docs/install/Fedora.md]: ths

**Pull request closed without merging**